real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/localizers/HighTrustGPSLocalizer.java

```diff
@@ -39,51 +39,66 @@ public HighTrustGPSLocalizer(){
     	buggyFrameGpsX = 0.0;
     	buggyFrameGpsY = 0.0;
         buggyFrameRotZ = 0.0;
+        buggySteeringAngle = 0.0;// wheel direction in buggy frame
```

Comment:
The canonical robotics  way of dealing with directions is to make frames right handed.  It is also typical to have x be forward for mobile robots, It is also common to have z be forward in the image frame.  I remember that Ian and I argued about this a few times.   

I think I  designed my code with z up x forward and y to the left.  So angles should be positive as you turn left.  You can change this but keep in mind that you need to revalidate that everything still works